graphql_api/types/flake_aggregates/flake_aggregates.py

```diff
@@ -0,0 +1,23 @@
+from ariadne import ObjectType
+
+flake_aggregates_bindable = ObjectType("FlakeAggregates")
+
+
+@flake_aggregates_bindable.field("flakeCount")
+def resolve_flake_count(obj, _) -> int:
+    return obj["flake_count"]
+
+
+@flake_aggregates_bindable.field("flakeRate")
+def resolve_flake_rate(obj, _) -> float:
+    return obj["flake_rate"]
+
+
+@flake_aggregates_bindable.field("flakeCountPercentChange")
```

Comment:
nit: would personally organize these in "bunches" like flakeCount together and then flakeRate together vs. all the counts and all the %'s

I could see either approach tho